TACC/tup-ui
# TUP-145: Add Secure Upload function

## Overview
Add Secure Upload iframe (same Box snippet used in old TUP).
## Related

- [TUP-145](https://jira.tacc.utexas.edu/browse/TUP-145)

## Testing

1. Under "Manage Account" click the "Secure Upload link"
2. Check that the Box iframe displays correctly.
3. Check that the "Back to Profile" link goes back to your profile page.

## UI
![image](https://user-images.githubusercontent.com/12601812/226732150-e78f1f09-f01e-4b56-9615-b5996c3809fd.png)
![image](https://user-images.githubusercontent.com/12601812/226751814-18a95121-94dc-4f9e-b5d0-fd1c9ffa279d.png)



## Notes
